package.json

```diff
@@ -13,7 +13,7 @@
     "request-promise": "4.2.6"
```

Comment:
Can we update the version number in `package.json` and `package-lock.json` to 1.0.8?
I'm guessing the current automated scripts are not doing that.